spec/controllers/base_controller_spec.rb

```diff
@@ -1,9 +1,9 @@
 require 'spec_helper'
 
 describe BaseController, :type => :controller do
-  let(:oc)    { mock_model(OrderCycle) }
-  let(:hub)   { mock_model(Enterprise, ready_for_checkout?: true) }
-  let(:order) { mock_model(Spree::Order, distributor: hub) }
+  let(:oc)    { double(:order_cycle, id: 1) }
+  let(:hub)   { double(:enterprise, ready_for_checkout?: true) }
+  let(:order) { double(:order, distributor: hub) }
```

Comment:
In this way we loose the method / column check on the stubbed model, did you try `instance_double(OrderCycle)`?
https://relishapp.com/rspec/rspec-rails/v/3-5/docs/model-specs/verified-doubles